avaframe/in3Utils/geoTrans.py

```diff
@@ -149,6 +150,64 @@ def resizeData(raster, rasterRef):
         return raster['rasterData'], rasterRef['rasterData']
 
 
+def remeshDEM(cfg, dem):
+    """ change DEM cell size by reprojecting on a new grid
+
+    Parameters
+    ----------
+    cfg : configParser
+        meshCellSizeThreshold : threshold under which no remeshing is done
+        meshCellSize : desired cell size
+
+    dem : dict
+        dem dictionary
+
+    Returns
+    -------
+    dem : dict
+        reprjected dem dictionary
+
+    """
+
+    cszThreshold = cfg.getfloat('meshCellSizeThreshold')
+    cszNew = cfg.getfloat('meshCellSize')
+    # read dem header
+    headerDEM = dem['header']
+    nColsDEM = headerDEM.ncols
+    nRowsDEM = headerDEM.nrows
+    xllcenter = headerDEM.xllcenter
+    yllcenter = headerDEM.yllcenter
+    cszDEM = headerDEM.cellsize
+    # remesh if input DEM size does not correspond to the computational cellSize
+    if np.abs(cszNew - cszDEM) > cszThreshold:
+        log.info('Remeshing the input DEM (of cell size %.2g m) to a cell size of %.2g m' % (cszDEM, cszNew))
+        x = np.linspace(0, (nColsDEM-1)*cszDEM, nColsDEM) + xllcenter
+        y = np.linspace(0, (nRowsDEM-1)*cszDEM, nRowsDEM) + yllcenter
+        z = dem['rasterData']
+        fInterp = sp.interpolate.RectBivariateSpline(x, y, np.transpose(z), ky=3, kx=3)
```

Comment:
I absolutely like the simplicity of using RectBivariateSpline, but I'm undecided wheter griddata would give more options (like bilinear, cubic etc interpolation). Question is: do we want those options? Or should we just put a comment beforehand that this would be the spot to add griddata if the need ever arises?

Reply:
you can here change the degree of the spline which ends up in kind of the same result I think

Reply:
Ok, then: mention in the docstring that RectBivar... is used, and that this would be the place to swap to another function like griddata or similar if the need arises